llvm/lib/IR/IntrinsicInst.cpp

```diff
@@ -237,8 +237,10 @@ void DbgAssignIntrinsic::setValue(Value *V) {
 }
 
 int llvm::Intrinsic::lookupLLVMIntrinsicByName(ArrayRef<const char *> NameTable,
-                                               StringRef Name) {
+                                               StringRef Name,
```

Comment:
This function seems to be misplaced in this file, in the sense it has nothing to do with `IntrinsicInst`. I propose we move it to Function.cpp other intrinsic name lookup related code resides (again, as a separate NFC).

Reply:
SG, I'll do that. I am also going see if adding an Intrinsics.cpp file instead makes sense.